docs/source/user-manual/links-to-item.md

```diff
@@ -27,7 +27,7 @@ Click then on the item labeled {guilabel}`Links to item`.
 You can see now a table displaying all links to the current item.
 
 ```{image} ../_static/user-manual/manage/link-to-items.png
-:alt: A panel captioned with "Links to My Summer Vacation - Day 2". A table with two columns (first column labeled with "linked by this item", second column labeled with "review state"). In the row appears a link titled "Links to My Summer Vacation - Day 1", because it is referencing the current item.
+:alt: A panel captioned with "Links to My Summer Vacation - Day 2". A table with two columns (first column labeled with "Linking this item", second column labeled with "review state"). In the row appears a link titled "Links to My Summer Vacation - Day 1", because it is referencing the current item.
```

Comment:
This summarizes the language that I would like to see used and adapted in the code for the panel. I probably missed exactly where the updates would be necessary in the code.

I think the row `Content that link to this item` can be merged with the heading above the table, into `Content that links to {ITEM_NAME}`.

```suggestion
:alt: A panel with the heading "Content that links to {ITEM_NAME}", followed by a table describing the content.
      The table has sections that show the content is linked by either a hyperlink or a reference.
      The first column is labeled "LINKED BY HYPERLINK" or "LINKED BY REFERENCE '{REF_NAME}'".
      The second and third columns are labeled "REVIEW STATE" and "TYPE".
```